runtime/oti/j9nonbuilder.h

```diff
@@ -335,6 +335,9 @@ typedef struct J9CfrError {
 	I_32 errorBsmIndex;
 	U_32 errorBsmArgsIndex;
 	U_32 errorCPType;
+	U_16 majorVersion;
+    U_16 minorVersion;
```

Comment:
Pls follow the same formatting convention as used in the file. i.e. tabs instead of spaces.